docs/_includes/header.html

```diff
@@ -0,0 +1,23 @@
+<header class="sticky-top bg-black">
+  <div class="navbar navbar-dark bg-dark navbar-expand-lg" aria-label="Global header of ODS Android">
+    <div class="container-xxl">
+      <!-- Orange brand logo -->
+      <div class="navbar-brand me-auto me-lg-4">
+        <span>
+          <img src="https://c.woopic.com/logo-orange.png" width="50" height="50" alt="ODS Android" loading="lazy">
```

Comment:
I think this is not the right icon, it should be the SVG version which contains the minified version at some breakpoints. Given the fact that there won't be too many users related to this documentation, you could try to use this one https://boosted.orange.com/docs/5.3/assets/brand/orange-logo.svg.
I know it's linked to a fixed 5.3 version, but this image is not in the Woopic CDN for now.